stock_product_last_usage_report/report/report_stock_lines_date.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018 Odoo, S.A.
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+from odoo import tools
+
+
+class ReportStockLinesDate(models.Model):
+    _name = "report.stock.lines.date"
+    _description = "Dates of Inventories and latest Moves"
+    _auto = False
+    _order = "date"
+    id = fields.Integer('Product Id', readonly=True)
+    product_id = fields.Many2one('product.product', 'Product', readonly=True,
+                                 index=True,
+                                 )
+    date = fields.Datetime('Date of latest Inventory', readonly=True)
+    move_date = fields.Datetime('Date of latest Stock Move', readonly=True)
+    active = fields.Boolean("Active", readonly=True)
+
+    @api.model_cr
+    def init(self):
+        tools.drop_view_if_exists(self.env.cr, self._table)
+        self.env.cr.execute("""
```

Comment:
You should put `# pylint: disable = E8103` before to avoid travis error.

Reply:
@lreficent No! The error means that you have to pass parameters as real arguments to execute() function. Look for code, there are plenty of examples.

Reply:
here it was passed with args and yet the pyling disable was needed: https://github.com/OCA/stock-logistics-warehouse/blob/10.0/stock_inventory_exclude_sublocation/models/stock_inventory.py#L55

Reply:
plus here what is being inyected is this: `self._table,`. Do you prefer to hardcode the table name above as odoo does (https://github.com/odoo/odoo/blob/9.0/addons/stock/report/report_stock.py#L23)?

Reply:
> here it was passed with args and yet the pyling disable was needed: https://github.com/OCA/stock-logistics-warehouse/blob/10.0/stock_inventory_exclude_sublocation/models/stock_inventory.py#L55

This one is a bad example as domain is a string 'injected' in query

Reply:
No. Just don't pass arguments as string injection

Reply:
> This one is a bad example as domain is a string 'injected' in query
I see, that's right. Then we can go with the args in the execute function :+1: